tests/framework/AnalyticModels/optimizing/beale.py

```diff
@@ -35,6 +35,8 @@ def run(self,Inputs):
     @ Out, None
   """
   self.ans = evaluate(self.x,self.y)
+  # test secondary variable passthrough
+  self.aux_ans = self.x + self.y
```

Comment:
please change "aux_ans" to "auxAns".

Reply:
this isn't a RAVEN variable, I don't think that applies here. We haven't applied code standards to ExternalModels at any point, I don't think ... maybe we need an issue for updating all external models in the repository.

Reply:
This is my suggestion, we better follow the standards when we want to push any codes into our repository.